docs/development.md

```diff
@@ -87,16 +84,14 @@ glide-vc -h
   Add new packages or subpackages to `glide.yaml` depending if you added whole
   new package as dependency or just new subpackage.
 
-2. Get new dependencies
-
-```bash
-glide update --strip-vendor
-```
+  Kedge vendors whole OpenShift and all its dependencies. (see comments in `glide.yaml` and `./scripts/vendor-openshift.yaml`)
+  It is possible that dependence you want to add is already in vendor as OpenShift dependency. If that is true, please make sure 
```

Comment:
`the dependency`
`in OpenShift as a vendored dependency`

Reply:
fixed